docs/faq/codex-wildbook-faq.md

```diff
@@ -3,3 +3,9 @@
 ## What's the difference between an Encounter and a Sighting?
 
 [Sightings](../introduction/sighting.md) in Wildbook are the same as in Codex - a group of animals at a related time and place. An animal from the Codex Animal tab is the same as a Wildbook [Encounter](../introduction/encounter.md) page.
+
+## The dates and times on my Encounters are different from the ones I reported in my Sightings.
+
+Codex used a time zone calculation that was not fully implemented in the front end. Wildbook does not use time zones and transferred the date and time data without consideration for it, interpreting all reported dates and times as UTC (time zone Z). We plan on adding [time zone support](https://github.com/WildMeOrg/Wildbook/issues/721) to Wildbooks as part of a multi-stage fix across releases. 
```

Comment:
"Wildbook does not use time zones and transferred the date and time data without consideration for it" sounds like we just didn't care or do anything, which isn't entirely accurate. We cared, it's just really hard to accurately migrate times that recognize time zones to times that don't

Reply:
" Wildbook does not use time zones and as a result, all transferred dates and times were interpreted as UTC (time zone Z)."